daemon/container_windows.go

```diff
@@ -61,13 +61,18 @@ func populateCommand(c *Container, env []string) error {
 
 	parts := strings.SplitN(string(c.hostConfig.NetworkMode), ":", 2)
 	switch parts[0] {
-
 	case "none":
 	case "default", "": // empty string to support existing containers
 		if !c.Config.NetworkDisabled {
 			en.Interface = &execdriver.NetworkInterface{
-				MacAddress: c.Config.MacAddress,
-				Bridge:     c.daemon.config.Bridge.VirtualSwitchName,
+				MacAddress:   c.Config.MacAddress,
+				Bridge:       c.daemon.config.Bridge.VirtualSwitchName,
+				PortBindings: c.hostConfig.PortBindings,
+
+				// TODO Windows. Include IPAddress. There already is a
```

Comment:
I would like re-iterate here that we should add the necessary capabilities in HCS to enable CNM for window as well so that we enable docker native networking functions like network scopes and service discovery across all docker platforms regardless of the OS. In that sense we should not add more properties via the execdriver path but instead work on HCS enhancements to move over to CNM. This is just a comment and should not block this PR.